packages/editor/src/client/atoms/descriptionListwithMessage.tsx

```diff
@@ -0,0 +1,23 @@
+import React, {ReactNode} from 'react'
+import {InfoMessage, InfoColor} from './infoMessage'
+import {DescriptionListItem} from './descriptionList'
+
+export interface DescriptionListWithMessageItemProps {
+  label?: ReactNode
+  children?: ReactNode
+  message?: ReactNode
```

Comment:
If this component is now meant specifically for messages, I think you should make the message required.

Reply:
That makes sense 👍